src/main/java/com/puppetlabs/jenkins/plugins/puppetgatling/gatling/PuppetGatlingBuildAction.java

```diff
@@ -4,26 +4,36 @@
 
 import java.util.*;
 
+import com.puppetlabs.jenkins.plugins.puppetgatling.PuppetGatlingProjectAction;
 import hudson.model.Action;
-import hudson.model.AbstractBuild;
+import hudson.model.Run;
+import io.gatling.jenkins.BuildSimulation;
+import jenkins.tasks.SimpleBuildStep;
 
 /**
  * Puppet Gatling Build Action
  *
  * @author Brian Cain
  */
-public class PuppetGatlingBuildAction implements Action {
+public class PuppetGatlingBuildAction implements Action, SimpleBuildStep.LastBuildAction {
```

Comment:
nit - does the `LastBuildAction` need to be qualified by `SimpleBuildStep` here?